### DISCUSSION TITLE
第3节课作业(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#343] [作业]
宝子们，第三次的作业交这里哦20班

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_3/lesson_3_homework.md

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/678603784

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135586743

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/678608161

**Comment by yalunren** [comment]
部署截图
<img width="968" alt="屏幕截图 2024-01-22 235738" src="https://github.com/InternLM/tutorial/assets/23251243/ca412dc2-d683-431e-8b6a-f3812673b33d">
<img width="963" alt="屏幕截图 2024-01-23 001330" src="https://github.com/InternLM/tutorial/assets/23251243/63eb9879-cfcd-456a-9166-e6049f867218">
